source/guide_openslides.rst

```diff
@@ -0,0 +1,208 @@
+.. highlight:: console
+
+.. author:: GodMod <godmod@eqdkp-plus.eu>
+.. tag:: lang-python
+.. tag:: django
+.. tag:: presentations
+.. tag:: web
+.. tag:: assemblies
+.. tag:: meetings
+.. tag:: audience-business
+
+.. sidebar:: About
+
+  .. image:: _static/images/openslides.svg
+      :align: center
+
+##########
+OpenSlides
+##########
+
+.. tag_list::
+
+OpenSlides is a free, web-based presentation and assembly system for managing and projecting agenda, motions, and elections of assemblies.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Python <lang-python>` and its package manager pip
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`Web Backends <web-backends>`
+
+License
+=======
+
+OpenSlides_ is released under the `MIT License`_.
+
+Prerequisites
+=============
+
+Your URL needs to be setup:
+
+::
+
+ [isabell@stardust ~]$ uberspace web domain list
+ isabell.uber.space
+ [isabell@stardust ~]$
+
+Installation
+============
+
+Step 1
+------
+Install the OpenSlides Python package:
+
+::
+
+ [isabell@stardust ~]$ pip3.9 install openslides --user
+ [...]
+ Running setup.py install for pyrsistent ... done
+ Running setup.py install for PyPDF2 ... done
+ Running setup.py install for roman ... done
+ Running setup.py install for websockets ... done
+ Running setup.py install for openslides ... done
+ [...]
+ [isabell@stardust ~]$
+
+Check if OpenSlides is installed by typing:
+
+::
+
+ [isabell@stardust ~]$ openslides --version
+ 3.3
+ [isabell@stardust ~]$
+
+Step 2
+------
+
+.. note::
+
+    OpenSlides is running on port 8000.
+
+.. include:: includes/web-backend.rst
+
+Step 3
+------
+Start the OpenSlides binary:
```

Comment:
Why is it necessary to start openslides manually before configuring the service? First doing the supervisord setup and then the user setup in the browser seems like an easier route to me, but maybe I am missing something.